src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -37,8 +56,80 @@ OtpUser preCreateHook(OtpUser user, Request req) {
         return super.preCreateHook(user, req);
     }
 
+    @Override
+    protected void buildEndpoint(ApiEndpoint baseEndpoint) {
+        LOG.info("Registering path {}.", ROOT_ROUTE + "verify");
+
+        // Add the api key route BEFORE the regular CRUD methods
+        ApiEndpoint modifiedEndpoint = baseEndpoint
+            .get(path(ROOT_ROUTE + ID_PATH + VERIFY_ROUTE)
+                    .withDescription("Request an SMS verification to be sent to an OtpUser's phone number.")
+                    .withPathParam().withName(ID_PARAM).withDescription("The id of the OtpUser.").and()
+                    .withResponseType(VerificationResult.class),
+                this::sendVerificationText, JsonUtils::toJson
+            )
+            .post(path(ROOT_ROUTE + ID_PATH + VERIFY_ROUTE + CODE_PATH)
```

Comment:
Remove `ROOT_ROUTE` from the path string to make this endpoint work as intended (see in `ApiController` [this](https://github.com/ibi-group/otp-middleware/blob/dev/src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java#L137) vs. [this](https://github.com/ibi-group/otp-middleware/blob/dev/src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java#L127))